go.md

```diff
@@ -30,42 +30,42 @@ func main() {
 ## Variables
 
 ### Declaring Variables
-```c
+```go
 var variable-name data-type
 ```
 ### Assigning value to variables
-```c
+```go
 variable-name := value // It's not mandatory to declare a variable when you use this.
 ```
 ### Example
-```c
-  var i int  // declaring int variable at functional level
-  j := 99 // short assignment without var declaration
+```go
+var i int  // declaring int variable at functional level
+j := 99 // short assignment without var declaration
 ```
 ## Data types
 
 ### 1. Numeric Data types
 
 #### Integer Data types
-| Data type | Description | Size|Range|
-|-----|-----|-----|----|
-|uint8|8-bit unsigned integer|1 byte|0 to 255|
+| Data type | Description | Size|Range|Alias|
+|-----|-----|-----|----|----|
+|uint8|8-bit unsigned integer|1 byte|0 to 255|byte|
```

Comment:
What is Alias?

Reply:
go type have alternate name for them. you can declare as `uint8` or `byte` it will have same type.

Reply:
Thanks, @beam41 could you please add `NA` for the column if Alias does not exist?

Reply:
Sure